Datenbank konfig wird nicht mehr gefunden nach update von 3.0.5 zu 3.1.5

Hallo,

ich habe Jverein über Arch AUR installiert. 
leider hat der Maintainer das lange nicht mehr aktualisiert. nach meinem Hinweis hat er es jetzt getan. 
aber irgendwie findet JVerein seine Datenbank konfig nicht mehr. 
Hat sich da was geändert?

Ich habe selbstverständlich vor dem Update ein Backup der DB gemacht. Und die Konfig sichert Jameica ja selbst...

Vielen Dank


..../jameica-mysql/cfg$ cat de.jost_net.JVerein.rmi.JVereinDBService.properties                                                                                                                                                                       database.driver=de.jost_net.JVerein.server.DBSupportMySqlImpl
database.driver.mysql.jdbcurl=jdbc\:mysql\://localhost\:3306/jverein?serverTimezone=Europe/Berlin&useUnicode\=Yes&characterEncoding\=ISO8859_1&allowPublicKeyRetrieval\=true
database.driver.mysql.username=jverein
database.driver.mysql.password=XXX
database.driver.mysql.scriptprefix=mysql-
`

`[Mon Sep 22 15:59:37 CEST 2025][ERROR][main][de.jost_net.JVerein.Einstellungen.<clinit>] Fehler beim laden der Einstellungen
java.rmi.RemoteException: error while getting database service; nested exception is: 
	de.willuhn.util.ApplicationException: Der Service "database" wurde nicht gefunden
	at de.jost_net.JVerein.Einstellungen.getDBService(Einstellungen.java:692)
	at de.jost_net.JVerein.Einstellungen.loadEinstellungen(Einstellungen.java:544)
	at de.jost_net.JVerein.Einstellungen.<clinit>(Einstellungen.java:121)
	at de.jost_net.JVerein.gui.boxes.MitgliedNextBGruppeChecker.<init>(MitgliedNextBGruppeChecker.java:63)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.ReflectAccess.newInstance(ReflectAccess.java:128)
	at java.base/jdk.internal.reflect.ReflectionFactory.newInstance(ReflectionFactory.java:304)
	at java.base/java.lang.Class.newInstance(Class.java:725)
	at de.willuhn.jameica.services.BeanService.get(BeanService.java:140)
	at de.willuhn.jameica.gui.boxes.BoxRegistry.getBoxes(BoxRegistry.java:63)
	at de.willuhn.jameica.gui.internal.views.Start.paint(Start.java:89)
	at de.willuhn.jameica.gui.internal.views.Start.bind(Start.java:76)
	at de.willuhn.jameica.gui.GUI$5.run(GUI.java:719)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:67)
	at de.willuhn.jameica.gui.GUI$6.run(GUI.java:916)
	at org.eclipse.swt.widgets.Synchronizer.syncExec(Synchronizer.java:183)
	at org.eclipse.swt.widgets.Display.syncExec(Display.java:5967)
	at de.willuhn.jameica.gui.GUI.startSync(GUI.java:912)
	at de.willuhn.jameica.gui.GUI.startView(GUI.java:596)
	at de.willuhn.jameica.gui.GUI.startView(GUI.java:531)
	at de.willuhn.jameica.gui.internal.action.Start.handleAction(Start.java:28)
	at de.willuhn.jameica.gui.Navigation.start(Navigation.java:394)
	at de.willuhn.jameica.gui.Navigation$MyActionListener.handleEvent(Navigation.java:489)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:91)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:5862)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1656)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1682)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1665)
	at org.eclipse.swt.widgets.Widget.notifyListeners(Widget.java:1379)
	at de.willuhn.jameica.gui.Navigation.select(Navigation.java:368)
	at de.willuhn.jameica.gui.GUI.loop(GUI.java:928)
	at de.willuhn.jameica.gui.GUI.init(GUI.java:335)
	at de.willuhn.jameica.system.Application.init(Application.java:145)
	at de.willuhn.jameica.system.Application.newInstance(Application.java:87)
	at de.willuhn.jameica.Main.main(Main.java:78)
Caused by: de.willuhn.util.ApplicationException: Der Service "database" wurde nicht gefunden
	at de.willuhn.jameica.system.ServiceFactory.lookup(ServiceFactory.java:227)
	at de.jost_net.JVerein.Einstellungen.getDBService(Einstellungen.java:687)
	... 35 more`

--- a/src/de/jost_net/JVerein/server/DDLTool/Updates/Update0479.java
+++ b/src/de/jost_net/JVerein/server/DDLTool/Updates/Update0479.java
@@ -92,6 +92,7 @@ public class Update0479 extends AbstractDDLUpdate
             break;
           case Types.CHAR:
           case Types.VARCHAR:
+          case Types.LONGVARCHAR:
             value = result.getString(i).replace("\n", "\\n").replace("\\",
                 "\\\\");
             break;
@@ -99,15 +100,43 @@ public class Update0479 extends AbstractDDLUpdate
           case Types.TINYINT:
           case Types.BIGINT:
           case Types.DOUBLE:
+          case Types.SMALLINT:
+          case Types.FLOAT:
+          case Types.NUMERIC:
+          case Types.DECIMAL:
             value = result.getObject(i).toString();
             break;
           case Types.BLOB:
           case Types.LONGVARBINARY:
+          case Types.BINARY:
+          case Types.VARBINARY:
+          case Types.NCHAR:
+          case Types.NVARCHAR:
             value = Base64.encode(result.getBytes(i));
             break;
+          // Alle restlichen Typen sind nicht implementiert
+          case Types.REAL:
+          case Types.TIME:
+          case Types.TIMESTAMP:
+          case Types.NULL:
+          case Types.OTHER:
+          case Types.JAVA_OBJECT:
+          case Types.DISTINCT:
+          case Types.STRUCT:
+          case Types.ARRAY:
+          case Types.CLOB:
+          case Types.REF:
+          case Types.DATALINK:
+          case Types.ROWID:
+          case Types.NCLOB:
+          case Types.SQLXML:
+          case Types.REF_CURSOR:
+          case Types.TIME_WITH_TIMEZONE:
+          case Types.TIMESTAMP_WITH_TIMEZONE:
           default:
-            String fehler = "Kann Einstellung nicht lesen, Type nicht implementiert: "
-                + m.getColumnType(i);
+            String fehler = "Kann Einstellung nicht lesen, Type "
+                + m.getColumnType(i) + " in Spalte " + col
+                + "nicht implementiert: ";
             Logger.error(fehler);
             throw new ApplicationException(fehler);
         }

--- a/src/de/jost_net/JVerein/server/DDLTool/Updates/Update0479.java
+++ b/src/de/jost_net/JVerein/server/DDLTool/Updates/Update0479.java
@@ -136,7 +136,7 @@ public class Update0479 extends AbstractDDLUpdate
           default:
             String fehler = "Kann Einstellung nicht lesen, Type "
                 + m.getColumnType(i) + " in Spalte " + col
-                + "nicht implementiert: ";
+                + " nicht implementiert: ";
             Logger.error(fehler);
             throw new ApplicationException(fehler);
         }

